gnocchi/storage/_carbonara.py

```diff
@@ -117,9 +118,15 @@ def _store_metric_measures(metric, timestamp_key, aggregation,
                                granularity, data, offset=None, version=3):
         raise NotImplementedError
 
-    @staticmethod
-    def _list_split_keys_for_metric(metric, aggregation, granularity,
+    def _list_split_keys_for_metric(self, metric, aggregation, granularity,
                                     version=3):
+        return set(map(
+            functools.partial(carbonara.SplitKey, sampling=granularity),
+            utils.to_timestamps(self._list_split_keys(
```

Comment:
```
-            utils.to_timestamps(self._list_split_keys(
-                metric, aggregation, granularity, version))))
+            self._list_split_keys(
+                metric, aggregation, granularity, version)))
```

i applied that and all the tests still pass which makes me think it's either not needed or not related to the changes in this commit.